tests/Fixer/FunctionNotation/FunctionDeclarationFixerTest.php

```diff
@@ -161,6 +171,141 @@ function foo() /* bar */
      * @param string      $expected
      * @param null|string $input
      *
+     * @dataProvider provideClosureFunctionSpacingNoneCases
+     */
+    public function testFixClosureFunctionSpacingNone($expected, $input = null)
```

Comment:
should not exist.
modify `testFix` to allow optional `array $configuration = null` and pass it in data provider